src/components/card/card.twig

```diff
@@ -1,32 +1,35 @@
-{% set tag_name = tag_name|default('article') %}
-{% set header_tag_name = header_tag_name|default('header') %}
-{% set footer_tag_name = footer_tag_name|default('footer') %}
+{% set _tag_name = tag_name|default('article') %}
+{% set is_sectioning = _tag_name in ['article', 'section'] %}
+{% set default_header_tag = is_sectioning ? 'header' : 'div' %}
+{% set default_footer_tag = is_sectioning ? 'footer' : 'div' %}
```

Comment:
This is a total nit, but in general, if I don't expect developers to use a property via `include` or `embed`, I lead its name with an underscore. Just helps me sort out which properties are "private" to the template versus which ones I expect users to actually manipulate.